src/tests/Common/scripts/bringup_runtest.sh

```diff
@@ -1,1325 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
Why is this being deleted?

Reply:
These scripts seem outdated and unused by anything in the repo.